.github/workflows/release.yml

```diff
@@ -0,0 +1,17 @@
+name: Release
+on:
+  release:
+    types:
+      - created
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Build
+        uses: ./.github/workflows/build.yml
```

Comment:
@andremalan not sure I'm doing it right (yet), but looks like there's now a syntax for re-using other workflows!